manifests/monitoring/kube-prometheus-stack/release.yaml

```diff
@@ -22,6 +22,8 @@ spec:
     grafana:
       sidecar:
         dashboards:
+          #label: grafana_dashboard
```

Comment:
Please remove these if they are not needed

Reply:
OK, squashed. 👍